views/home/index.html

```diff
@@ -34,15 +34,15 @@ <h1 class="page-header">Welcome to the SCD Cloud</h1>
                     <h1 class="page-header">Infrastructure</h1>
                     <img src="/assets/images/rack.png" style="width:170px;float:right;margin-top:-15px" />
                     <p>
-                        The cloud is based on <a href="http://opennebula.org">OpenNebula</a> for the virtualisation and <a href="http://ceph.com">CEPH</a> for the storage. For the hardware, we use
+			The cloud is based on <a href="http://{{ cloudPlatform }}.org" target="_blank">{{ cloudPlatform | title }}</a> for the virtualisation and <a href="http://ceph.com" target="_blank">CEPH</a> for the storage. For the hardware, we use
```

Comment:
Can you remove the hardware specs, they're likely to now be wrong and no-one should really care anyway.
Perhaps the storage platform should also be templated?